docs/UserGuide.adoc

```diff
@@ -37,6 +37,7 @@ What's different from AddressBook-Level1:
 * Support for storing address (`a/`) and tags (`t/`)
 * Support for marking a contact detail as 'private' (`pa/`) (`pe/`) (`pp/`)
 * View details of a person (`view` : shows non-private details), (`viewall` : shows all details)
+* View current calendar year every time a command is prompted from the user
```

Comment:
I remember you mentioned your intention in the tutorial for doing this. 
While this enhancement earns you full marks for the LO, I am not sure if you got the full experience of working with an intermediate-size codebase.